### PR TITLE
Activity log awerb/authority/ready question playback

### PR DESCRIPTION
**Pull request by wheelsandcogs**
~Will need rebasing once https://github.com/UKHomeOffice/asl-pages/pull/916 is merged, this is a continuation of that work.~

### CODE REVIEW

File: pages/task/read/views/components/activity-log.jsx [hunk 1]
```diff
@@ -1,43 +1,45 @@
...
 import classnames from 'classnames';
 import get from 'lodash/get';
-import { Link, Snippet, Markdown } from '@asl/components';
+import { Link, Snippet, Markdown, Inset } from '@asl/components';
 import { dateFormat } from '../../../../../constants';
 import format from 'date-fns/format';
+import PplDeclarations from './ppl-declarations';
 
-const getProfileLink = ({ id, name, establishmentId, asruUser }) => {
+function ProfileLink({ id, name, establishmentId, asruUser }) {
```
Comment (joefitter) [inline pages/task/read/views/components/activity-log.jsx:9]: I like this, if it looks like a component, make it a component 👍

File: pages/common/assets/sass/style.scss [hunk 2]
```diff
@@ -512,3 +517,11 @@ details {
     vertical-align: top;
   }
 }
+
+#submitted-version {
+  .declarations {
+    .question span {
+      font-weight: bold;
```
Comment (joefitter) [inline pages/common/assets/sass/style.scss:524]: `<strong>` makes them bold without having to add extra styles

Comment (wheelsandcogs) [reply pages/common/assets/sass/style.scss:524]: We re-use the component in two places with different styling - in the activity log it's designed as non-bold and in the "Latest submission" section it's designed as bold, so this was the easiest way to get both.

File: pages/task/read/views/components/activity-log.jsx (hunk 1 above)
Comment (lennym) [reply pages/task/read/views/components/activity-log.jsx:9]: I agree. Nice.

File: pages/common/assets/sass/style.scss (hunk 2 above)
Comment (joefitter) [reply pages/common/assets/sass/style.scss:524]: seems legit

**Pull request merged**